api/s2n.h

```diff
@@ -67,6 +69,12 @@ extern int s2n_error_get_type(int error);
 
 struct s2n_config;
 struct s2n_connection;
+struct s2n_security_policy;
+
+extern const struct s2n_security_policy *S2N_SECURITY_POLICY_DEFAULT;
+extern const struct s2n_security_policy *S2N_SECURITY_POLICY_DEFAULT_TLS13;
+extern const struct s2n_security_policy *S2N_SECURITY_POLICY_20170210;
+extern const struct s2n_security_policy *S2N_SECURITY_POLICY_20190801;
```

Comment:
Why make these security policies public? Is there a more useful name we could give them to indicate why a customer would use them?

Reply:
I would argue these are the most useful names since they correlate with the cipher preferences they are meant to deprecate :) What should they be named?

I can also just remove them.

Reply:
So the idea here is customers can pin to the versioned (dated) security policies to avoid having "default" change unexpectedly. But if we aren't really releasing this yet, then there is no reason to expose any of them. This is a larger conversation about whether this PR is really something we should be doing until a builder is put together.

Reply:
I think it's a good approach to incrementally roll this out, rather than wait for the builder. We know we need the APIs that you've implemented here and I think everyone is ok with them. But we can be as conservative in the process as we'd like. In this case, we can always export more policies but we can never take them out. So if we start with default, we can at least merge this.

Reply:
Then we should leave the actual policies so customers can pin to something that won't change. This is the same approach we take with cipher preferences. Why would customers expect something different here? Why would customers migrate to this new API if they have to switch to a policy which may change without warning?

Reply:
I think the assumption is that people won't migrate to this new API until the builder exists. There's not really an incentive to.